cub/device/device_segmented_radix_sort.cuh

```diff
@@ -64,6 +64,9 @@ CUB_NAMESPACE_BEGIN
  * DeviceSegmentedRadixSort shares its implementation with DeviceRadixSort. See
  * that algorithm's documentation for more information.
  *
+ * @par Segments are not required to span `num_items`. Ranges that are not 
+ * covered by segments are guaranteed not to be modified. 
+ *
```

Comment:
I think we can make the statement even stronger. 

What do you think?
```suggestion
 * @par Segments are not required to be contiguous. Any element of input(s) or output(s) 
 * outside the specified segments will not be accessed nor modified.  
 *
```

Reply:
"not be access nor modified" is a bit redundant since you can't modify something without accessing it, but sometimes a little redundancy helps make a point clearer on first read.

Reply:
But you can access memory without modifying it, so that statement is both stronger and more correct

Reply:
Right, I was explaining my decision to say "not be accessed nor modified" instead of just saying "not be accessed".